Xsoar mirroring bug (#24233)
demisto/content: Packs/XSOARmirroring/ReleaseNotes/2_0_12.md

#### Integrations
##### XSOAR Mirroring

> - Added a parameter 'Mirror Playbook ID' which allows to choose if mirroring the playbookId or not.

Reviewer: ```suggestion
- Added the boolean `Drop Playbook ID` parameter. When set to true, the receiving instance will run the default playbook matching the incident type (if one exists).
```

> - Added the boolean `Drop Playbook ID` parameter. When set to true, the receiving instance will run the default playbook matching the incident type (if one exists).

Reviewer: ```suggestion
- Added the boolean `Mirror Playbook ID` parameter. When set to `false`, the receiving instance will run the default playbook matching the incident type (if one exists). Default is `true`, preseving backwards compatibility.
```

> - Added the boolean `Mirror Playbook ID` parameter. When set to `false`, the receiving instance will run the default playbook matching the incident type (if one exists). Default is `true`, preseving backwards compatibility.

Reviewer: ```suggestion
- Added the boolean `Mirror Playbook ID` parameter. Default is `true`, preseving backwards compatibility. When set to `false`, the receiving instance will run the default playbook matching the incident type (if one exists). 
```
